Library/Formula/voms.rb

```diff
@@ -12,14 +11,19 @@ class Voms < Formula
     sha256 "01ed7c6185bbd2609e958065c1d58bdd8617abccdcfd3d8875faae532d327dea" => :mavericks
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
+  depends_on "maven" => :build
   depends_on "openssl"
 
   def install
-    system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "mvn", "package"
+    system "tar", "-xf", "target/voms-clients.tar.gz"
+    share.install "voms-clients/share/java"
+    man.mkpath
```

Comment:
You can remove this line. `man5.install` below will create the required directories.